src/06-loops.js

```diff
@@ -61,8 +83,15 @@ function rangeEveryOther(min, max) {}
  *  containsWhileLoop([ "left", "up", "right" ], "down");
  *  //> false
  */
-function containsWhileLoop(array, target) {}
-
+function containsWhileLoop(array, target) {
+let i=0;  
+while (i < array.length){
+  if (array[i] === target){
+    return true
+} i++
+}
+return false 
```

Comment:
Lets take a quick look at our indentation here - we get flat towards the end it's harder to read.